src/components/SearchMultiSelectInput/index.tsx

```diff
@@ -253,31 +261,66 @@ function SearchMultiSelectInput<
         [name, onChange],
     );
 
+    const optionListRendererParams = useCallback(
+        (key, option) => ({
+            contentRendererParam: optionRendererParams,
+            option,
+            optionKey: key,
+            focusedKey,
+            contentRenderer: Option,
+            onClick: handleOptionClick,
+            onFocus: setFocusedKey,
+            readOnly,
+            disabled,
+            optionContainerClassName: _cs(selectedOptionContainerClassName, styles.selectedItem),
+        }),
+        [
+            readOnly,
+            disabled,
+            focusedKey,
+            handleOptionClick,
+            optionRendererParams,
+            selectedOptionContainerClassName,
+        ],
+    );
+
     return (
-        <SelectInputContainer
-            {...otherProps}
-            name={name}
-            options={realOptions}
-            optionsPending={optionsPending}
-            optionsFiltered={searchInputValue?.length > 0}
-            optionKeySelector={keySelector}
-            optionRenderer={Option}
-            optionRendererParams={optionRendererParams}
-            optionContainerClassName={styles.optionContainer}
-            onOptionClick={handleOptionClick}
-            valueDisplay={valueDisplay}
-            onClear={handleClear}
-            searchText={searchInputValue}
-            onSearchTextChange={handleSearchValueChange}
-            onDropdownShownChange={handleChangeDropdown}
-            focused={focused}
-            onFocusedChange={setFocused}
-            focusedKey={focusedKey}
-            onFocusedKeyChange={setFocusedKey}
-            persistentOptionPopup
-            nonClearable={false}
-            hasValue={isDefined(value) && value.length > 0}
-        />
+        <>
```

Comment:
We should either wrap with div or SelectInputContainer should support some prop for this

Reply:
@AdityaKhatri, @tnagorra what do you suggest we do? Is it a good idea if we pass it as a child to SelectInputContainer?

Reply:
@frozenhelium what do you think?

Reply:
@AdityaKhatri  I have no idea what we're trying to achieve here

Reply:
Should the SelectInputContainer or InputContainer support showing a component just below it?

We wouldn't need a wrapper if it is supported.

Reply:
@AdityaKhatri how do we handle this ? We can use this component in l`ead copy to other projects`.